docs/website/netcdf-java/tutorial/NetcdfFile.html

```diff
@@ -43,7 +43,7 @@ <h2 >Opening a <a>NetcdfFile</a>
 <h3>Using ToolsUI to browse the metadata of a dataset </h3>
 
 <p>The NetCDF Tools User Interface (aka ToolsUI) is a program for browsing and debugging NetCDF files. You can start it from the command line, or
-  <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">launch it from webstart</a>. </p>
+  <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">launch it from webstart</a>. </p>
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.